README.md

```diff
@@ -36,7 +36,7 @@ doesn't comply with the license).
 
 ### Apache header:
 
-    Copyright 2019 Expedia, Inc
+    Copyright 2019 Expedia Group, Inc.
```

Comment:
No, the lawyers specifically said this has to remain `Expedia, Inc`. Please ensure this is so in all downstream projects.